modules/oadp-usecase-include-ca-cert-backup.adoc

```diff
@@ -0,0 +1,274 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-enable-ca-cert.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-backup-restore-ca-cert_{context}"]
+= Backing up an application by including a self-signed CA certificate
+
+The `s3.openshift-storage.svc` service by {odf-short}, uses a TLS certificate that is signed with the self-signed service CA. To prevent a `certificate signed by unknown authority` error, you must include a self-signed CA certificate in the backup storage location (BSL) section of the Data Protection Application (DPA). In this use case, you:
```

Comment:
```suggestion
The `s3.openshift-storage.svc` service, provided by {odf-short}, uses a TLS certificate that is signed with the self-signed service CA. To prevent a `certificate signed by unknown authority` error, you must include a self-signed CA certificate in the backup storage location (BSL) section of the Data Protection Application (DPA). In this use case, you:
```